esmvaltool/utils/cmorizers/obs/cmorize_obs.py

```diff
@@ -245,19 +240,17 @@ def _cmor_reformat(config, obs_list):
                             dataset, reformat_script)
 
                 # call the ncl script
-                _run_ncl_script(in_data_dir,
-                                out_data_dir,
-                                run_dir,
-                                dataset,
-                                reformat_script,
-                                config['log_level'])
-            elif os.path.isfile(reformat_script_root + '.py'):
-                py_reformat_script = reformat_script_root + '.py'
+                _run_ncl_script(in_data_dir, out_data_dir, run_dir, dataset,
+                                reformat_script, config['log_level'])
```

Comment:
```suggestion
                                reformat_script, config['log_level'],)
```
and `yapf -i` again to get a nicer formatting